R/utils.R

```diff
@@ -10,6 +10,7 @@ is.dependency <- function(x) inherits(x, "dash_dependency")
 is.output <- function(x) inherits(x, "output")
 is.input <- function(x) inherits(x, "input")
 is.state <- function(x) inherits(x, "state")
+is.event <- function(x) is.dependency(x) && inherits(x, "event")
```

Comment:
This was accidental and should not have been committed. It has since been reversed.